src/yaml.ts

```diff
@@ -135,8 +140,13 @@ export function parseYAML(contents: string): {
   // rather than the ASTs (via the `stringify` function we return).
   const topLevelTokens = [...new yaml.Parser().parse(contents)];
   const documents = [
-    ...new yaml.Composer({ keepSourceTokens: true }).compose(topLevelTokens),
+    ...new yaml.Composer({
+      keepSourceTokens: true,
+    }).compose(topLevelTokens),
   ];
+  // Doing an extra parse so we can char offsets to line/columns for errors.
```

Comment:
can you just pass `lineCounter.addNewline` to `new Parser()` above like parseDocument does internally?